cmd/tools/gen-accessors.go

```diff
@@ -0,0 +1,300 @@
+// Copyright 2017 The go-github AUTHORS. All rights reserved.
+//
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
```

Comment:
I think you should add:

```
// +build ignore
```

It won't affect go generate functionality since the .go file is being `go run` directly.

But it'll prevent a poorly named and unneccessary `tools` binary from being placed in users `$GOPATH/bin` directory if they install all packages in go-github repository.

In fact, is there an advantage in placing this file in a separate directory, instead of just inside `github` folder, the same folder as the `github` package where it's being used?